pyccel/commands/console.py

```diff
@@ -60,7 +60,7 @@ def pyccel(files=None, mpi=None, openmp=None, openacc=None, output_dir=None, com
     # ... backend compiler options
     group = parser.add_argument_group('Backend compiler options')
 
-    group.add_argument('--language', choices=('fortran', 'c', 'python'), help='Generated language')
+    group.add_argument('--language', choices=('fortran', 'c', 'python', 'ccuda'), help='Generated language')
```

Comment:
There should be an equivalent change in the `epyccel` docstring

Reply:
[8e318c5](https://github.com/pyccel/pyccel-cuda/pull/12/commits/8e318c5644c41aaa4e7d721345179bc5fcf3d297)